watch.sh

```diff
@@ -1,4 +1,4 @@
-SASSC_COMMAND='sassc sass/all.sass static/assets/all.css --style compressed'
+SASSC_COMMAND='sassc sass/all.scss static/assets/all.css --style compressed'
```

Comment:
This change needs to be reflected in circleci/config.yml as well.

Reply:
I updated the circleci/config.yml

Reply:
thanks